src/compute-client/src/metrics.rs

```diff
@@ -86,7 +87,7 @@ impl ComputeControllerMetrics {
             command_message_bytes_total: metrics_registry.register(metric!(
                 name: "mz_compute_command_message_bytes_total",
                 help: "The total number of bytes sent in compute command messages.",
-                var_labels: ["instance_id", "replica_id", "command_type"],
```

Comment:
Had to remove the command type from the byte count metrics because the CTP metrics don't support it. I think that's fine, we still have the command count metrics.